static/js/containers/ProfileImage_test.js

```diff
@@ -1,26 +1,31 @@
 /* global SETTINGS: false */
-import '../global_init';
-
 import React from 'react';
 import { mount } from 'enzyme';
 import { assert } from 'chai';
 import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
 import getMuiTheme from 'material-ui/styles/getMuiTheme';
+import { Provider } from 'react-redux';
+import _ from 'lodash';
 
 import ProfileImage from './ProfileImage';
 import IntegrationTestHelper from '../util/integration_test_helper';
+import ProfileImageUploader from '../components/ProfileImageUploader';
+import * as api from '../lib/api';
+import { startPhotoEdit, updatePhotoEdit } from '../actions/image_upload';
+import { USER_PROFILE_RESPONSE } from '../constants';
 
 describe('ProfileImage', () => {
-  let helper;
+  let helper, sandbox, updateProfileImageStub, defaultProps;
 
-  const renderProfileImage = (profile, editable=true) => (
+  const renderProfileImage = (editable=true, props = defaultProps) => (
```

Comment:
Can we set a default `profile` below instead and use `{}` in place of `defaultProps`?